modules/ipi-install-setting-cluster-node-hostnames-dhcp.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+//  *installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ipi-install-setting-cluster-node-hostnames-dhcp_{context}"]
+= Setting the cluster node hostnames through DHCP
+
+On {op-system-first} machines, the hostname is set through NetworkManager. 
+By default, the machines obtain their hostname through DHCP. 
+If the hostname is not provided by DHCP, set statically through kernel arguments, or another method, it is obtained through a reverse DNS lookup. 
+Reverse DNS lookup occurs after the network has been initialized on a node and can take time to resolve. Other system services can start prior to this and detect the hostname as `localhost` or similar. 
+You can avoid this by using DHCP to provide the hostname for each cluster node.
```

Comment:
Maybe specify what `this` is referring to. `You can avoid this delay in assigning hostnames by using...`

Reply:
I'm inclined to leave this. I think it flows ok from the previous sentence. 
The alternative
"You can prevent other system services from starting and detecting the hostname as `localhost` or similar by using DHCP to provide the hostname for each cluster node." 
seems a bit unwieldy and repetitive